R/pairwise-comparisons.R

```diff
@@ -328,31 +258,42 @@ pairwise_comparison_one_group <- function(scores,
 #' @inheritParams pairwise_comparison
 #' @param name_model1 character, name of the first model
 #' @param name_model2 character, name of the model to compare against
+#' @param one_sided Boolean, default is `FALSE`, whether two conduct a one-sided
+#' instead of a two-sided test to determine significance in a pairwise
+#' comparison.
+#' @param test_type character, either "non_parametric" (the default) or
+#' "permutation". This determines which kind of test shall be conducted to
+#' determine p-values.
+#' @param n_permutations numeric, the number of permutations for a
+#' permutation test. Default is 999.
 #' @author Johannes Bracher, \email{johannes.bracher@@kit.edu}
 #' @author Nikos Bosse \email{nikosbosse@@gmail.com}
+#' @keywords internal
 
 compare_two_models <- function(scores,
                                name_model1,
                                name_model2,
                                metric,
-                               test_options,
-                               by) {
-
+                               one_sided = FALSE,
+                               test_type = c("non_parametric", "permutation"),
```

Comment:
Possibly add a warning if `test_type == "non_parametric"` but `n_permutations` is provided.